roles/openshift_logging/defaults/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
-openshift_logging_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
-openshift_logging_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default(
+  'registry.access.redhat.com/openshift3/ose-' if openshift_deployment_type == 'openshift-enterprise'
+  else 'docker.io/openshift/origin-') }}"
```

Comment:
@ewolinetz I don't see why we couldn't go down that route for the simple use cases. It might get a bit tricky with defaults that depend on more than a single variable, think a combination of deployment type and particular versions. I don't think we'd want to maintain a complex matrix of variable files.